CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 - PR #32 Python API first pass
 - PR #37 Python __init__.py package design
 - PR #38 Add __init__.py empties to resolve issue with PYTHONPATH
+- PR #97 Drop `cython` from run requirements
```

Comment:
This should be under bug fixes or improvements. It's not a new feature.

Reply:
Ah thought it was under improvements. Sorry. Will move. Thanks.